openwisp-qa-check

```diff
@@ -60,6 +67,36 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  package='stylelint'
+  if which stylelint > /dev/null; then
+    stylelint verbose $(find . -type f -name "*.css") &&
+      echo "SUCCESS: Stylelint check successful!" ||
+      {
+        echoerr "ERROR: Stylelint check failed! Hint: did you forget running openwisp-qa-format? You may need some manual fixing."
```

Comment:
I think we can change it. to "ERROR: Stylelint check failed! Hint: did you forget running openwisp-qa-format? You may need to do some manual fixing."